docs/releases/1.2.rst

```diff
@@ -77,7 +77,7 @@ Minor features
  * WagtailRedirectMiddleware can now ignore the query string if there is no redirect that exactly matches it
  * Order of URL parameters now ignored by redirect middleware
  * Added SQL Server compatibility to image migration
- * Added classnames to Wagtail rich text editor buttons to aid custom styling
+ * Added ``classnames`` to Wagtail rich text editor buttons to aid custom styling
```

Comment:
Let's say `Added ``class`` attributes` instead. (Putting `classnames` in monospace implies that there's a variable / identifier named `classnames`, which there isn't...)

Reply:
Ahh, assumed it was!